src/Card/index.js

```diff
@@ -1,50 +1,107 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './style.css';
 import { BiEdit } from 'react-icons/bi';
 import { AiOutlineSave } from 'react-icons/ai';
 import { ImCancelCircle } from 'react-icons/im'
 
-const card = props => {
+const Card = props => {
+  const [cardState, setCardState] = useState({
+    checked: false,
+    isEditable: false,
+    data: {
+      headerData: props.headerData,
+      bodyData: props.bodyData
+    },
+    tempData: {
+      headerData: props.headerData,
+      bodyData: props.bodyData
```

Comment:
It could actually be blank from the initializing and be set while switching to edit mode.